### DISCUSSION TITLE
BoxedWine does not work

### DISCUSSION THREAD

**Discussion by ghost** [DustinBrett/daedalOS#344] [General]
boxxedwine doesnt launch any exe files and says it cant find the wine files in system32

**Comment by DustinBrett** [comment]
Hi @ElytriGit,

I just tested BoxedWine and it actually is working, but there are caveats.

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/ce2ffeb9-9a32-4ba8-a835-cf971c31cd85)

BoxedWine is very picky and only works under certain circumstances. The app needs to be Win 16/32, not x64 in any way. Also the exe needs to have all it's dependent files included, so if there are any dll's and such, it needs to be loaded as a zip and not directly as the exe.

If you take a screenshot of the error BoxedWine says we can figure out why it didn't work for you.

**Reply by ghost** [reply]
oh ok, i was testing lightweight games like "five nights at frieddys" and "osu!"(2007 version) it didnt work with for me, i will try other apps, also this whole web os is amazing!

**Reply by DustinBrett** [reply]
Thanks! Ya those games may be a bit advanced or require dependencies. Here is BoxedWine's demo site (http://www.boxedwine.org/search-apps/) with some of the apps that work. Typically the app needs to be very portable and have no installer.

**Comment by DustinBrett** [comment]
I've moved this to general discussion because the BoxedWine integration with my site is working, afaik. Any issues with what it can run are specific to BoxedWine. If you post a screenshot and/or share the exe, I can try and see why it won't run, but it would be something to do with BoxedWine which I don't have the ability to change. I plan to upgrade it when the new version comes out which was discussed here https://github.com/danoon2/Boxedwine/issues/45#issuecomment-1279992581
